pylintrc

```diff
@@ -46,14 +43,17 @@ disable=
     consider-iterating-dictionary,
     consider-merging-isinstance,
     consider-using-dict-comprehension,
+    consider-using-dict-items,
```

Comment:
any reasons for adding these? is it to avoid more fixing in some files?

Reply:
Its to avoid fixing this line: https://github.com/aristanetworks/ansible-cvp/blob/ebcc2ebee58adc93d47938be219e5f397e6dc1c5/ansible_collections/arista/cvp/plugins/module_utils/device_tools.py#L796

Reply:
But I agree to your point, I will add `fixme` to places where we need to fix things later

Reply:
ok makes sense!